### ISSUE TITLE
Скрывать введенные пароли в настройках

### ISSUE THREAD

**Issue by Pro100x3mal** [yumata/lampa#178]
Большая просьба реализовать возможность скрывать введенные пароли в настройках Лампы. Возможно отдельной опцией в Остальных настройках.
Это касается авторизации для CUB, авторизации для torrserver, api-key для jackett.
Все-таки приватность хочется соблюдать.
Часто в сети делятся скриншотами с настройками, где видны личные данные пользователей

**Comment by totavok8** [comment]
когда походите авторизацию в кубе пароль скрывается. он отображается только в процессе ввода что является логичным

**Comment by Pro100x3mal** [comment]
@totavok8 да, с этим так и есть. А вот с авторизацией в торрсервере и апи ключ для джакета - нет. Хотелось бы и там реализовать, например заменить уже введенные и примененные данные на символы «*»

**Comment by burjuyz** [comment]
присоединяюсь! не хватает этого
